### DISCUSSION TITLE
How to open specific camera view in android axis companion app

### DISCUSSION THREAD

**Discussion by stanford-scs** [AxisCommunications/discussions#209] [General]
I'm trying to tie together a bunch of axis camera with a number of alarm sensors.  When certain alarm sensors are activated, I send out an android notification.  One of the actions for the notification is to open the axis companion app with `app://com.axis.companion`.  However, this only shows the last camera viewed.  I would like to open the specific camera that is most relevant to the location of the particular alarm sensor that was activated.  When cameras have multiple views, as with the Axis P3719, I would like to open up the most relevant view.

Is there an intent or deep link that I can use to open up a specific camera view?  In an ideal world, I could offer an action to open the archived video to a specific time, so that someone who clicks the notification later can view the exact archived video from the time at which the alarm was tripped.  However, I would settle for just opening the relevant view for the sensor.

**Comment by vivekatoffice** [comment]
AXIS Companion has been upgraded to AXIS Camera Station Edge, offering enhanced features and improved performance. Learn more and download AXIS Camera Station Edge [here](https://www.axis.com/products/axis-camera-station-edge).
